torch/csrc/jit/register_special_ops.cpp

```diff
@@ -1,11 +1,11 @@
+#include <aten/src/ATen/ExpandUtils.h>
```

Comment:
Do you mind commenting here what was actually changed in this file? It's hard to review with all the formatting changes.

Even better would be to stack the formatting changes in a separate PR under this one 😉

Reply:
Yea this was an annoying format change. I'll try to separate as a different commit

Reply:
Here is the view: https://github.com/pytorch/pytorch/pull/19445/files/1c6957781c314fa073c1004ee96811e88e66e560..26b29d7dd8221510aeb254056a10989976369c41